demo/admin/src/pages/EditPage.tsx

```diff
@@ -204,3 +213,11 @@ export const EditPage: React.FC<Props> = ({ id, category }) => {
         </EditPageLayout>
     );
 };
+
+const StatusButton = styled(IconButton)`
+    width: 50px;
+    height: 50px;
+    border-radius: 0;
+    color: ${({ theme }) => theme.palette.common.white};
+    border-left: 1px solid #2e3440;
+`;
```

Comment:
If this styling is always necessary, we should add something to the library to ease adding a custom action.